CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+# OP-TEE - version 1.1.0
+
+
+## New features
+
+* Softfloat library: floating point support is now available in 32bits TA.
+
+* Support running 64-bits TA: on ARMv8 platform, TA can be compiled in
```

Comment:
ARM is usually quite picky about differentiate between different architectures, so please change to ARMv8-A instead of only ARMv8.